schemas/url.yml

```diff
@@ -95,6 +95,14 @@
         the query field exists with an empty string. The `exists`
         query can be used to differentiate between the two cases.
 
+    - name: extension
+      level: extended
+      type: keyword
+      short: File extension from the original request url.
+      description: >
+        The field contains the file extension from the original request url.
+        The file extension is only set if it exists, as not every url has a file extension.
```

Comment:
Please insert an additional blank line here, to have this as two paragraphs in the docs